File: java/oop/AccountPolymorphismApp/src/com/techlabs/account/CurrentAccount.java
package com.techlabs.account;

public class CurrentAccount extends Account{
	public CurrentAccount(int acno,String  acntname,double balance){
		super(acno,acntname,balance);
	}
	public void withdraw(double amt){
		if((this.balance-amt)<(-5000)){
			System.out.println("Cannot withdraw....Limit Exceed");
		}else{
			this.balance=this.balance-amt;
		}

	}

}

File: dotNet/csharp/AccountPolymorphismApp/AccountPolymorphismApp/CurrentAccount.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AccountPolymorphismApp
{
    class CurrentAccount : Account
    {
        public CurrentAccount(int acno, String acntname, double balance):base(acno,acntname,balance)
        {
            
        }
        public override void Withdraw(double amt)
        {
            if ((this._balance - amt) < (-5000))
            {
                Console.WriteLine("Cannot withdraw....Limit Exceed");
            }
            else
            {
                this._balance = this._balance - amt;
            }
        }
    }
}
